File: C++ Programs/Searching Algorithms/Linear Search/CPP_Linear_Search_Count_Occurrences-main/Count Occurance.cpp
#include<iostream>
using namespace std;
int main()
{
	bool isTrue=false;
	int arr[5];
	for(int i=0; i<=4; i++){
		cout<<"Enter Element: ";
		cin>>arr[i];
	}
	
	int search, count=0;
	cout<<"Enter value to search: ";
	cin>>search;
	for(int i=0; i<=4; i++){
		if(search == arr[i]){
			count++;
			isTrue=true;
		}
	}
	if(isTrue){
		cout<<"Count: "<<count<<endl;
		return 0;
	}
	else
	cout<<"Element Not Found!"<<endl;
}

File: Java Programs/Linear Search/Java_Linear_Search_Count_Occurrences-main/CountOccurance.java
import java.util.Scanner;

public class CountOccurance {
    public static void main(String[] args) {
        int count=0;
        boolean found=false;
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter Value of N: ");
        int n=sc.nextInt();
        int arr[]= new int[n];
        for(int i=0; i<n; i++){
            System.out.print("Enter element: ");
            arr[i]=sc.nextInt();
        }
        System.out.print("Enter element to find Occurance: ");
        int search=sc.nextInt();
        for(int i=0; i<n; i++){
            if(search == arr[i]){
                count++;
                found=true;
            }
        }
        if(found){
            System.out.println("Occurance: "+count);
        }
        else{
            System.out.println("Not Found!");
        }
    }
}
